lib/android/app/src/main/java/com/reactnativenavigation/views/BottomTabs.java

```diff
@@ -29,6 +30,11 @@ public void setBadge(int bottomTabIndex, String badge) {
         setNotification(badge, bottomTabIndex);
     }
 
+    public void setBadgeColor(@ColorInt Integer color) {
+        if (color == null) return;
```

Comment:
You should also check if the current badge color equals `color` and return if true. Otherwise the BottomTabs view will be recreated unnecessarily which results in noticeable artifacts.

Reply:
@SCadilhac You're right, merging :)